lib/models/widget_model.dart

```diff
@@ -89,6 +91,45 @@ class DashboardWidget {
     };
   }
 
+  bool isWarningWidget() {
+    if (content.containsKey(DashboardWidget.WIDGET_STATUS_KEY)) {
+      final WidgetStatus widgetStatus =
+      EnumToString.fromString(WidgetStatus.values, content[DashboardWidget.WIDGET_STATUS_KEY]);
+      return hasWarningStatus(widgetStatus);
+    } else {
+      return false;
+    }
+  }
+
+  bool hasWarningStatus(WidgetStatus widgetStatus) {
+    return widgetStatus == WidgetStatus.CHECKBOX_UNKNOWN ||
+        widgetStatus == WidgetStatus.UNKNOWN ||
+        widgetStatus == WidgetStatus.UNSTABLE;
+  }
+
+  bool isErrorWidget() {
```

Comment:
Please drop `Widget` from the method name and be more `DRY` so instead calling `widget.isErrorWidget()` we will call `widget.isError()` - please apply for similar cases as well.

Reply:
done